var/spack/repos/builtin/packages/nek5000/package.py

```diff
@@ -46,9 +46,9 @@ class Nek5000(Package):
     tags = ['cfd', 'flow', 'hpc', 'solver', 'navier-stokes',
             'spectral-elements', 'fluid']
 
-    version('17.0.0-beta2', git='https://github.com/Nek5000/Nek5000.git',
-        commit='b95f46c59f017fff2fc19b66aa65a881085a7572')
-    version('develop',      git='https://github.com/Nek5000/Nek5000.git',
+    version('17.0',    git='https://github.com/Nek5000/Nek5000.git',
+        commit='469daf94d3f9aa3ba9d258d8eee9ebde6893a702', preferred=True)
```

Comment:
No need for preferred, it’ll be used by default

Reply:
Fixed, Thanks !!